lexicons/app/bsky/feed/gate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.gate",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["post", "createdAt"],
+        "properties": {
+          "post": {"type": "string", "format": "at-uri"},
+          "allow": {
+            "type": "array",
+            "maxLength": 5,
+            "items": {
+              "type": "union",
+              "refs": [
+                "#mentionRule",
+                "#followingRule",
+                "#listRule"
+              ]
+            }
+          },
+          "createdAt": {"type": "string", "format": "datetime"}
+        }
+      }
+    },
+    "mentionRule": {
+      "type": "object",
+      "description": "Allow replies from actors mentioned in your post."
+    },
+    "followingRule": {
+      "type": "object",
+      "description": "Allow replies from actors you follow."
+    },
+    "listRule": {
+      "type": "object",
+      "description": "Allow replies from actors on a list.",
+      "required": ["list"],
+      "properties": {
+        "list": {"type": "ref", "ref": "com.atproto.repo.strongRef"}
```

Comment:
We could alternately make this a weak ref.  I went back and forth on this one.

Reply:
yeah I think a weak ref 🤔 similar to a `listitem`, we only provide the `listUri` not the cid. 

the list cid also doesn't relate to the contents of the list. so it would really only be an integrity check against the list displayname/avatar/etc